src/vivarium_public_health/disease/model.py

```diff
@@ -87,20 +124,14 @@ def _get_default_initial_state(self):
             raise DiseaseModelError("Disease model must have exactly one SusceptibleState.")
         return susceptible_states[0].state_id
 
-    def time_step_handler(self, event):
-        self.transition(event.index, event.time)
-
-    def time_step__cleanup_handler(self, event):
-        self.cleanup(event.index, event.time)
-
     def get_state_weights(self, pop_index, prevalence_type):
         states = [s for s in self.states
-                  if hasattr(s, f'{prevalence_type}_data') and getattr(s, f'{prevalence_type}_data') is not None]
+                  if hasattr(s, f'{prevalence_type}') and getattr(s, f'{prevalence_type}') is not None]
```

Comment:
Actually important.  I removed the 'data' suffix from the prevalence look up tables since it was not totally true.